packages/android_alarm_manager/example/android/app/src/main/java/io/flutter/plugins/androidalarmmanagerexample/Application.java

```diff
@@ -3,7 +3,7 @@
 import io.flutter.app.FlutterApplication;
```

Comment:
Is the entire `Application.java` file still necessary in V2 embedding?

Reply:
@bkonyi

Reply:
The README says that V2 should handle this via reflection. Is it safe to just delete this file?

Reply:
@bkonyi my read is that this file is still needed (since we have one application for both the (new)FlutterActivity and the EmbeddingV1Activity.java). So this file looks fine to me.